aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -282,65 +236,61 @@ dotnet ef database update
+```
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-  ```console
-  SqliteException: SQLite Error 1: 'no such table: Movie'.
-  ```
+```console
+SqliteException: SQLite Error 1: 'no such table: Movie'.
+```
 
 ---
-  You probably missed the [migrations step](#migration).
 
-* Test the **Create** page. Enter and submit data.
+Test the **Create** page. Enter and submit data.
 
-  > [!NOTE]
-  > You may not be able to enter decimal commas in the `Price` field. To support [jQuery validation](https://jqueryvalidation.org/) for non-English locales that use a comma (",") for a decimal point and for non US-English date formats, the app must be globalized. For globalization instructions, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/4076#issuecomment-326590420).
+> [!NOTE]
+> You may not be able to enter decimal commas in the `Price` field. To support [jQuery validation](https://jqueryvalidation.org/) for non-English locales that use a comma (",") for a decimal point and for non US-English date formats, the app must be globalized. For globalization instructions, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/4076#issuecomment-326590420).
 
-* Test the **Edit**, **Details**, and **Delete** pages.
+Test the **Edit**, **Details**, and **Delete** pages.
 
 ## Dependency injection in the controller
 
-# [Visual Studio](#tab/visual-studio)
-
 Open the *Controllers/MoviesController.cs* file and examine the constructor:
 
 <!-- l.. Make copy of Movies controller (or use the old one as I did in the 3.0 upgrade) because we comment out the initial index method and update it later  -->
 
 [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie22/Controllers/MC1.cs?name=snippet_1)]
 
-The constructor uses [Dependency Injection](xref:fundamentals/dependency-injection) to inject the database context (`MvcMovieContext`) into the controller. The database context is used in each of the [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) methods in the controller.
-
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie22/Controllers/MC1.cs?name=snippet_1)]
+In the preceding code:
 
-The constructor uses [Dependency Injection](xref:fundamentals/dependency-injection) to inject the database context (`MvcMovieContext`) into the controller. The database context is used in each of the [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) methods in the controller.
+* The constructor uses [Dependency Injection](xref:fundamentals/dependency-injection) to inject the database context `MvcMovieContext` into the controller.
+* The database context is used in each of the [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) methods in the controller.
 
 ### Use SQLite for development, SQL Server for production
```

Comment:
This needs to be hidden from VS by a tab. It's too much of a distraction for SQl Server users.

All the manual stuff that was removed by scaffolding needs to be added back in, but this time, in lieu of 
# Create a database context class
change that to 
# Examine the generated database context class

Then under the new examine section on the DB context, you can add the SQL in production to the VSC/VSmac tabs.

Reply:
@Rick-Anderson,   

Putting together your comments around putting the scaffolding sections back in as "Examine" sections, I think I have that structured the way you want right now in the PR. 

However the instruction:  "Then under the new examine section on the DB context, you can add the SQL in production to the VSC/VSmac tabs." makes me worry I don't have it right.  What I ended up with would not have VSC/VSMac tabs at that point, unless I have the tabs in the wrong organization now.

Could you take a look and see if I have the structure right and you envisioned it, and  if so, specify exactly what line where you want the SQL in production section.  It is not clear to me, sorry to ask for clarification.  Handy review link: [Internal Review - Adding a Model](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?view=aspnetcore-5.0&branch=pr-en-us-21521&tabs=visual-studio)

Reply:
The scaffolded code is the same except for startup and the connection string - right? Use tabs where you need them.

Reply:
> specify exactly what line where you want the SQL in production section. 

I took care of that in #21610

Reply:
Back to this.  I followed what you did in #21610.   Just a note though: I could not move the 4 line repeated text from the SQLite for dev... section to an include file I ended up repeating the 4 lines for VS Code and VS mac.   Oddly I if I removed those 4 lines, whether replacing them with an include fill or not, it would not merge.  I tested many times trying various ways to get it to allow a merge and it came down to removing text on lines 323 to 326 would result in a "Cannot merge commit" error.  I finally had to give in and just repeat the 4 lines between the two tabs, and worry about moving it to an include file (if that is absolutely necessary) after this PR.